CONTRIBUTING.md

```diff
@@ -75,8 +74,19 @@ either column and find a ticket you like. Create a branch from the HEAD of
 the `canon` branch, add your name to the [CONTRIBUTORS file][contributors],
 commit that change and open a PR:
 
-* Include "WIP" in the title or make a [draft pull request][draftpr]
-* Reference the issue you want to work on in the body of the PR.
+- Include "WIP" in the title or make a [draft pull request][draftpr]
+- Reference the issue you want to work on in the body of the PR.
+
+The editable installation mode implies that the source code of PursuedPyBear being installed is available in a local directory. Once PursuedPyBear is installed in editable mode, users expect that changes to the PursuedPyBear python code in the local source tree become effective without the need of a new installation step. To do an editable installation:
+
+    mkdir ppb
+    python3 -m venv ppb-venv
+    source ppb-venv/bin/activate
+    cd ppb-venv
+    git clone https://github.com/ppb/pursuedpybear.git
+    cd pursuedpybear
+    pip install --upgrade -r requirements.txt
```

Comment:
This should reference `requirements-test.txt` instead. The `requirements.txt` is fed into the setup configuration so you get for free as part of the pip install.

Reply:
ok great, I will make the change. Thanks!